SS14.Launcher/Views/MainWindow.xaml.cs

```diff
@@ -31,6 +32,10 @@ public MainWindow()
         _content = (MainWindowContent) Content!;
 
         ReloadTitle();
+
+#if DEBUG
+        this.AttachDevTools();
+#endif
```

Comment:
This should be done in `InitializeComponent()` according to [Avalonia's docs](https://docs.avaloniaui.net/docs/guides/implementation-guides/developer-tools). Is it not?

Reply:
My bad It is indeed, must have messed up when I first tried to use it. Reverted.